### DISCUSSION TITLE
Test notifications in Docker?

### DISCUSSION THREAD

**Discussion by ammacdonald3** [jdholtz/auto-southwest-check-in#155] [Q&A] [answered]
Hi, are there any ways to test notifications when running this in Docker? This app has been a godsend so far, but I feel like I'm flying blind without notifications. Hoping to confirm that they work before my next flight.

**Comment by jdholtz** [comment, marked as answer]
Hey. Glad you like it! To test the notifications, you can just append the `--test-notifications` flag as an argument to pass into the image (make sure it’s the last argument in the docker run command). 

Additionally, you can also just run the image with your flight/account details and it should notify you that it scheduled your flights if the URLs are correct (also make sure your notification level is 1 if you do it this way).

**Reply by ammacdonald3** [reply]
Thanks so much - that worked! The Apprise documentation seemed to indicate that it could identify the Telegram {chat_id} if the {bot_token} was provided, but it only worked for me when I explicitly included the {chat_id} in the config file too. Just including this info in case it helps anyone else. This is an incredible project; thanks again!
